### PR TITLE
[DE] Update `Medals`; add `Gameplay/Full_combo` and `Medals/Legacy_beatmap_packs`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
The diff of this PR isn't that big in reality. It mostly is due to the listing of songs in the `Legacy Beatmap Packs` article.

As a note aside, I've left the names of medal types (as well as the names of beatmap packs) in their original name, because the website doesn't offer localisation for that and I think it would be inconvenient for someone searching for the translated version and not finding anything.

I also didn't add any "adjust link when German article is available" todos, as I normally do, because there were just too many links in the article. Just need to carefully look for linked headings when adding/updating a new/existing article.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Medals/Legacy_beatmap_packs/de.md [hunk 1]
```diff
@@ -0,0 +1,406 @@
+# Originale Beatmap-Pakete
+
+Beatmap-Pakete sind ein integraler Bestandteil der osu!-Community seit [ihrer ersten Bekanntmachung](https://osu.ppy.sh/community/forums/topics/1853) von ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) vor langer Zeit im Jahr 2008, in dem einige der Pakete mit bestimmten [Errungenschaften](/wiki/Medals) verbunden waren. Als osu! und seine Community jedoch über die Jahre wuchsen, hat sich das Mapping-Meta im Allgemeinen seit seinen Anfängen stark verändert und es wurde — durch große Nachfrage der Öffentlichkeit — [am 16. Januar 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) eine umfassende Überarbeitung bei zahlreichen Beatmap-Paketen vorgenommen, um diese in ihrer Gesamtheit auf dem neusten Stand zu halten.
```
Comment (respektive) [inline wiki/Medals/Legacy_beatmap_packs/de.md:3]:
```suggestion
Beatmap-Pakete sind ein integraler Bestandteil der osu!-Community seit [ihrer ersten Bekanntmachung](https://osu.ppy.sh/community/forums/topics/1853) von ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) vor langer Zeit im Jahr 2008, in dem einige der Pakete mit bestimmten [Errungenschaften](/wiki/Medals) verbunden waren. Als osu! und seine Community jedoch über die Jahre wuchsen, hat sich die Mapping-Meta im Allgemeinen seit ihren Anfängen stark verändert und es wurde — durch große Nachfrage der Öffentlichkeit — [am 16. Januar 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) eine umfassende Überarbeitung bei zahlreichen Beatmap-Paketen vorgenommen, um diese in ihrer Gesamtheit auf dem neusten Stand zu halten.
```
Pretty sure it should be `die Meta` though I couldn't find any German definitions for this, but at least from my experience this is the most common.

File: wiki/Medals/Legacy_beatmap_packs/de.md [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+- [Zircon - Dirt Devil (OC ReMix)](https://osu.ppy.sh/beatmapsets/14205)
+- [Shoji Meguro - Reach Out to the Truth (Instrumental)](https://osu.ppy.sh/beatmapsets/14458)
+- [Masashi Hamauzu - Hope Given "Dance of the Dog's Howl"](https://osu.ppy.sh/beatmapsets/16669)
+- [Ken Nakagawa - Cyclone](https://osu.ppy.sh/beatmapsets/17373)
+- [Crush 40 - Sonic Heroes](https://osu.ppy.sh/beatmapsets/21836)
+- [Anamanaguchi - Scott Pilgrim Anthem](https://osu.ppy.sh/beatmapsets/23073)
+
+## Trivia
+
+- Die Beatmap [Loituma - Ievan Polkka](https://osu.ppy.sh/beatmapsets/2463), welche im Internet Pack (Volume 1) enthalten war, war berüchtigt dafür, in die Sektion [approved](/wiki/Beatmap/Category#approved) gedrückt worden zu sein, auch wenn sie nur aus einem einzigen Spinner bestand. Obwohl die Beatmap kurze Zeit später [vom dem Approved Beatmap Listing entfernt wurde](https://osu.ppy.sh/community/forums/posts/149328), blieben ihre Überreste trotzdem im Internet Pack (Volume 1).
```
Comment (respektive) [inline wiki/Medals/Legacy_beatmap_packs/de.md:403]:
```suggestion
- Die Beatmap [Loituma - Ievan Polkka](https://osu.ppy.sh/beatmapsets/2463), welche im Internet Pack (Volume 1) enthalten war, war berüchtigt dafür, in die Sektion [Approved](/wiki/Beatmap/Category#approved) verschoben worden zu sein, auch wenn sie nur aus einem einzigen Spinner bestand. Obwohl die Beatmap kurze Zeit später [vom dem Approved Beatmap Listing entfernt wurde](https://osu.ppy.sh/community/forums/posts/149328), blieben ihre Überreste trotzdem im Internet Pack (Volume 1).
```

File: wiki/Medals/de.md [hunk 3]
```diff
@@ -1,74 +1,59 @@
...
 
-Zur Zeit gibt es 16 dieser Erfolge, welche sich nochmal in 4 Untergruppen aufteilen lassen:
+Alle Beatmaps müssen im [osu!](/wiki/Game_mode/osu!)-Spielmodus bezwungen werden, um sich für das Abzeichen zu qualifizieren, sofern nicht anders angegeben. Die Verwendung von entweder [Mods zur Verringerung der Schwierigkeit](/wiki/Game_modifier#difficulty-reduction) oder [Mods zur Erhöhung der Schwierigkeit](/wiki/Game_modifier#difficulty-increase) ist erlaubt.
 
-- Video Game Pack
-- Anime Pack
-- Internet! Pack
-- Rhythm Game Pack
+Es ist jedoch erwähnenswert, dass der Inhalt aller Video Game Packs, Anime Packs, Internet! Packs und Rhythm Game Packs sich seit [ihrer ersten Bekanntmachung im Jahr 2008](https://osu.ppy.sh/community/forums/topics/1853) dramatisch verändert haben. Siehe [originale Beatmap-Pakete](/wiki/Medals/Legacy_beatmap_packs) für die Liste dieser Beatmap-Pakete in ihrem alten/originalen Zustand.
```
Comment (respektive) [inline wiki/Medals/de.md:11]:
```suggestion
Es ist jedoch erwähnenswert, dass der Inhalt aller Video Game Packs, Anime Packs, Internet! Packs und Rhythm Game Packs sich seit [ihrer ersten Bekanntmachung im Jahr 2008](https://osu.ppy.sh/community/forums/topics/1853) dramatisch verändert hat. Siehe [originale Beatmap-Pakete](/wiki/Medals/Legacy_beatmap_packs) für die Liste dieser Beatmap-Pakete in ihrem alten/originalen Zustand.
```

File: wiki/Medals/Legacy_beatmap_packs/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Medals/Legacy_beatmap_packs/de.md:3]: My initial thought was to go with the neutral article, but this is fine as well.

After some small internet search ~~(and excluding the new facebook lol)~~, I've found the following things:
- [A word definition on wiktionary](https://de.wiktionary.org/wiki/meta-), however this isn't helpful here
- Deepl actually translates `the game meta` with `die Spiel-Meta`

Comment (respektive) [reply wiki/Medals/Legacy_beatmap_packs/de.md:3]: Yeah i found the wiktionary one as well and Duden has it too, but both are for prefixes, so not really helpful.

File: wiki/Medals/de.md [hunk 4]
```diff
@@ -1,74 +1,59 @@
...
-- Das Emblem wird erst ausgehändigt, wenn alle Lieder aus dem Pack geschafft wurden.
-  - Der Rekord muss hochgeladen worden sein. Ansonsten kann es sein, dass die Beatmap ein weiteres Mal gespielt werden muss.
-- Bei alten Maps ist das Timing/Offset oftmals falsch. Mit dem `Offset Wizard` in den Einstellungen kann das Timing/Offset korrigiert werden.
+*Siehe auch: [Spielmodifikationen](/wiki/Game_modifier)*
 
-#### Trivial
+**Mod Introduction Medaillen** (zu Dt. Mod-Vorstellung) sind Medaillen, die beim Abschießen von [qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked) oder [approved](/wiki/Beatmap/Category#approved) Beatmaps mit der entsprechenden [Spielmodifikation (Mod)](/wiki/Game_modifier) überreicht werden. Diese Medaillen sollen neue Spieler dazu ermutigen, jede Mod auszuprobieren und zu entdecken, was das Spiel zu bieten hat. Bedenke jedoch, dass wenn Spieler mit mehreren Spielmodifikationen zur gleichen Zeit spielen, der Spieler **nicht** berechtigt ist, *irgendeine* der Medaillen für die Spielmodifikationen für dieses Spiel zu gewinnen.
```
Comment (respektive) [inline wiki/Medals/de.md:25]:
```suggestion
**Mod Introduction Medaillen** (zu Dt. Mod-Vorstellung) sind Medaillen, die beim Abschließen von [qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked) oder [approved](/wiki/Beatmap/Category#approved) Beatmaps mit der entsprechenden [Spielmodifikation (Mod)](/wiki/Game_modifier) überreicht werden. Diese Medaillen sollen neue Spieler dazu ermutigen, jede Mod auszuprobieren und zu entdecken, was das Spiel zu bieten hat. Bedenke jedoch, dass wenn Spieler mit mehreren Spielmodifikationen zur gleichen Zeit spielen, der Spieler **nicht** berechtigt ist, *irgendeine* der Medaillen für die Spielmodifikationen für dieses Spiel zu gewinnen.
```
When missing one letter changes the whole meaning of a word :sweat_smile:

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Medals/Legacy_beatmap_packs/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Medals/Legacy_beatmap_packs/de.md:403]:
```suggestion
- Die Beatmap [Loituma - Ievan Polkka](https://osu.ppy.sh/beatmapsets/2463), welche im Internet Pack (Volume 1) enthalten war, war berüchtigt dafür, in die Sektion [Approved](/wiki/Beatmap/Category#approved) verschoben worden zu sein, auch wenn sie nur aus einem einzigen Spinner bestand. Obwohl die Beatmap kurze Zeit später [von der Approved-Beatmap-Auflistung entfernt wurde](https://osu.ppy.sh/community/forums/posts/149328), blieben ihre Überreste trotzdem im Internet Pack (Volume 1).
```
Maybe sounds a bit better? 🤔

File: wiki/Medals/de.md (hunk 3 above)
Comment (The-Last-Cookie) [reply wiki/Medals/de.md:11]:
```suggestion
Es ist jedoch erwähnenswert, dass sich der Inhalt aller Video Game Packs, Anime Packs, Internet! Packs und Rhythm Game Packs seit [ihrer ersten Bekanntmachung im Jahr 2008](https://osu.ppy.sh/community/forums/topics/1853) dramatisch verändert hat. Siehe [originale Beatmap-Pakete](/wiki/Medals/Legacy_beatmap_packs) für die Liste dieser Beatmap-Pakete in ihrem alten/originalen Zustand.
```
Sounds better to me when reading it out loud.

File: wiki/Medals/de.md (hunk 4 above)
Comment (The-Last-Cookie) [reply wiki/Medals/de.md:25]: Adjusting to lazer *pew pew* 😝

File: wiki/Medals/de.md (hunk 3 above)
Comment (respektive) [reply wiki/Medals/de.md:11]: Yeah, that's better.

**Review by respektive — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**